libs/ui-components/src/components/form/TextListField.tsx

```
@@ -0,0 +1,88 @@
+  name: string;
+  isLoading?: boolean;
+  addButtonText?: string;
+  helperText?: React.ReactNode;
+  onChangeCallback?: (newList: string[], hasErrors: boolean) => void;
+  canEdit?: boolean;
+  focusAddOnStart?: boolean;
+};
+
+const maxWidth = '16ch';
+
+const TextListField = ({ name, onChangeCallback, addButtonText, focusAddOnStart, helperText }: TextListFieldProps) => {
```

ESLint

[error] 'focusAddOnStart' is defined but never used.

(@typescript-eslint/no-unused-vars)